tiledb/api/c_api/enumeration/enumeration_api.cc

```diff
@@ -191,8 +194,11 @@ CAPI_INTERFACE(
     const void* offsets,
     uint64_t offsets_size,
     tiledb_enumeration_t** enumeration) {
+  auto memory_tracker = ctx->context().resources().create_memory_tracker();
+  memory_tracker->set_type(tiledb::sm::MemoryTrackerType::ENUMERATION_CREATE);
```

Comment:
These can't be here since they're not guaranteed to not throw and this is outside our exception boundary which means if they ever did throw it'd be an instant segfault/abort.

Reply:
Does this mean I place these in a try catch block?

Reply:
Nope, you just need to change `api_entry_context` to `api_entry` or whatever and then create them inside the implementation of `tiledb::api::tiledb_enumeration_alloc` that starts on line 41.

Reply:
Done!